src/components/playlistCard/PlaylistCard.js

```diff
@@ -0,0 +1,38 @@
+import React from "react";
+import "./PlaylistCard.scss";
+import { Link } from "react-router-dom";
+import PlaylistImage from "../../assets/playlistImage.png";
+
+function PlaylistCard({ data }) {
+  return (
+    <div>
+      <Link className="playlistcard__link" to={`/playlist/${data.id}`}>
+        <div className="playlistcard">
+          <div className="playlistcard__thumbnail">
+            {data.images[0] ? (
```

Comment:
Add optional chaining to avoid crash if property is missing
```suggestion
            {data?.images[0] ? (
```